lib/OpenQA/Worker.pm

```diff
@@ -383,29 +383,9 @@ sub configure_cache_client {
     $client->ua->inactivity_timeout($ENV{OPENQA_WORKER_CACHE_SERVICE_CHECK_INACTIVITY_TIMEOUT} // 10);
 }
 
-# sleeps for the specified number of seconds
-# note: It was supposed to "sleep" for the specified number of seconds while actually running the worker's event
-#       loop started via exec() to keep processing events (like job cancellation). However, this caused problems
-#       (see poo#96710). Hence we're going back to using a normal sleep here.
-sub delay ($self, $delay) { sleep $delay }
-
-sub stop_event_loop ($self) {
```

Comment:
I'm wondering if it would have been useful to keep this helper so it can be "counted" in unit tests? To check that we stop correctly 🤔

Reply:
If we fail to stop the event loop within a test we would actually have a test which just hangs forever in any case. So it is not very useful. The only purpose of this function was to unset `$self->{_resume_loop}` which is now not need anymore (and hence doesn't need to be tested either).